rita_client/src/lib.rs

```diff
@@ -69,10 +71,11 @@ impl Default for Args {
 /// and does not need to be specified.
 pub fn get_client_usage(version: &str, git_hash: &str) -> String {
     format!(
-        "Usage: {} [--config=<settings>] [--platform=<platform>] [--future]
+        "Usage: {} [--config=<settings>] [--platform=<platform>] [--save_interval=<time>] [--future]
```

Comment:
arguments are pretty hard for us to modify on the routers, much better to remove this and use a config value.